### PR TITLE
Prepare release letter for 20200921

### PR DESCRIPTION
**Pull request by tkilias**

### CODE REVIEW

File: doc/changes/changes-20200803.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  * [script-languages-release@#57](https://github.com/exasol/script-languages-release/issues/57): Simplify the customization of flavors for users
+  * [script-languages-release@#23](https://github.com/exasol/script-languages-release/issues/23): Add script to install specific version of a R packages
+  * [script-languages@#127](https://github.com/exasol/script-languages/pull/127): Add more robust package install scripts which support package versions
+* Exslsct:
+  * [script-languages-release@#42](https://github.com/exasol/script-languages-release/pull/42): Add command to generate language activation statement
+
+## Refactorings
+
+* Exaslct:
+  * [script-languages-release@#46](https://github.com/exasol/script-languages-release/pull/46): Reorganizing exaslct and extracting the test environment to [exasol/integration-test-docker-environment](https://github.com/exasol/integration-test-docker-environment)
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes-20200803.md:44]:
```suggestion
  * [script-languages-release@#46](https://github.com/exasol/script-languages-release/pull/46): Reorganize exaslct and extract the test environment to [exasol/integration-test-docker-environment](https://github.com/exasol/integration-test-docker-environment)
```

File: doc/changes/changes-20200803.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  * [script-languages@#108](https://github.com/exasol/script-languages/pull/108): Add cleanup method to exaslct tasks and scripts to run exaslct tests
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes-20200803.md:45]:
```suggestion
  * [script-languages@#108](https://github.com/exasol/script-languages/pull/108): Add a cleanup method to exaslct tasks and scripts to run exaslct tests
```

File: doc/changes/changes-20200803.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  * [script-languages@#98](https://github.com/exasol/script-languages/pull/98): Pass docker socket to test container
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes-20200803.md:46]:
```suggestion
  * [script-languages@#98](https://github.com/exasol/script-languages/pull/98): Pass a docker socket to the test container
```

File: doc/changes/changes-20200803.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  * [script-languages@#142](https://github.com/exasol/script-languages/pull/142): Extract and refactored functionalities for performance testing into exatest
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes-20200803.md:47]:
```suggestion
  * [script-languages@#142](https://github.com/exasol/script-languages/pull/142): Extract and refactor functionalities for performance testing into exatest 
```

File: doc/changes/changes-20200803.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  * [script-languages-release@#42](https://github.com/exasol/script-languages-release/pull/42): Add command to generate language activation statement
+
+## Refactorings
+
+* Exaslct:
+  * [script-languages-release@#46](https://github.com/exasol/script-languages-release/pull/46): Reorganizing exaslct and extracting the test environment to [exasol/integration-test-docker-environment](https://github.com/exasol/integration-test-docker-environment)
+  * [script-languages@#108](https://github.com/exasol/script-languages/pull/108): Add cleanup method to exaslct tasks and scripts to run exaslct tests
+  * [script-languages@#98](https://github.com/exasol/script-languages/pull/98): Pass docker socket to test container
+  * [script-languages@#142](https://github.com/exasol/script-languages/pull/142): Extract and refactored functionalities for performance testing into exatest 
+  * [script-languages@#105](https://github.com/exasol/script-languages/pull/105): Fix reference to luigis FrozenOrderedDict and pin dependency versions in pipfile
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes-20200803.md:48]:
```suggestion
  * [script-languages@#105](https://github.com/exasol/script-languages/pull/105): Fix a reference to luigis FrozenOrderedDict and pin dependency versions in pipfile
```

File: doc/changes/changes-20200803.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+However, most importantly, it greatly simplifies the customization of the script-language containers. Each flavor now provides package lists under the directory `flavors/<flavor-name>/flavor_customization/packages`. Addtional packages entered into these lists get automatically installed when building the flavor. Furthormore, we provide detailed instrunctions how to do other kinds of modifications to the container by changing the Dockerfile at `flavors/<flavor-name>/flavor_customization/Dockerfile`. Please find more information about this, in our [Readme](https://github.com/exasol/script-languages-release#how-to-customize-an-existing-flavor).
+ 
+## Bug Fixes
+
+* UDFs:
+  * [script-languages-release@#67](https://github.com/exasol/script-languages-release/issues/67): Fix error in Python3 UDFs when emitting only a dataframe with one Timestamp column 
+  * [script-languages-release@#48](https://github.com/exasol/script-languages-release/pull/48): Fix Python3 UDF bug with reset after get_dataframe reaches the end of the iterator
```
[the remark this answers is not in the document]
Comment (tkilias) [reply doc/changes/changes-20200803.md:12]: yes, obvious a type, but the correction should be:

`Fix Python3 UDF bug during reset after get_dataframe reaches the end of the iterator`

File: doc/changes/changes-20200803.md (hunk 3 above)
Comment (tkilias) [reply doc/changes/changes-20200803.md:46]: `Pass the docker socket to the test container`

**Pull request merged**